lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -413,10 +413,9 @@ def add_3ds(post, options)
             post[:threeDS2RequestData] = { deviceChannel: device_channel, notificationURL: three_ds_2_options[:notification_url] }
           end
 
-          if options.has_key?(:execute_threed)
-            post[:additionalData][:executeThreeD] = options[:execute_threed]
-            post[:additionalData][:scaExemption] = options[:sca_exemption] if options[:sca_exemption]
-          end
+          post[:additionalData][:executeThreeD] = options[:execute_threed] || false
```

Comment:
If `executeThreeD` is sent to Adyen as false, it will override the `:threed_dynamic` choice made by the customer, and 3DS will not be performed. Up to this point, this has functioned like so:
- `execute_threed: true` definitely do 3DS
-  `threed_dynamic: true` include some additonal fields to indicate to Adyen to check the transaction against my preset rules. Perform 3DS only if it breaks my rules.
- neither true = the additional fields are not included, and `executeThreeD` is null. This third time is the only time we should be setting `executeThreeD` as false.